auto_backup/README.rst

```diff
@@ -0,0 +1,99 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Automated backups
+===========
```

Comment:
Please fix the syntax according to https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst